clients/nginxservice.go

```diff
@@ -1,12 +1,26 @@
 package clients
 
 import (
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/arm"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+type SvcInfo struct {
```

Comment:
What's the point of this struct? All the information in here seems to be redundant information that you can grab off the k8s service?

Reply:
I kept this struct in case I need it in the future if I'm deploying other services. But I will delete it for now because I'll can refer to previous commits.